frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -84,15 +84,15 @@ const Campaigns = () => {
     {
       name: 'Created At',
       render: (campaign: Campaign) => (
-        <Moment format="LLL">{campaign.createdAt}</Moment>
+        <Moment format="MMM DD YYYY, HH:mm">{campaign.createdAt}</Moment>
```

Comment:
Probably need to change the other functions that use 'LLL' as well, for consistency eg
https://github.com/opengovsg/postmangovsg/blob/8e5d63030bfc1bcfdb1dc71878feeee208358c63/frontend/src/components/common/export-recipients/ExportRecipients.tsx#L81 
https://github.com/opengovsg/postmangovsg/blob/8e5d63030bfc1bcfdb1dc71878feeee208358c63/frontend/src/classes/Campaign.ts#L98